agent/fw_lumen.c

```diff
@@ -36,6 +36,8 @@ static int nr_lumen_name_the_wt(const char* name TSRMLS_DC,
     } else {
       path += 1;
     }
+  } else {
+    path = name;
```

Comment:
This is needed otherwise the `path` is `NULL` when `strip_base` is set to `false`